chemprop/data/scaffold.py

```diff
@@ -71,8 +71,8 @@ def scaffold_split(data: MoleculeDataset,
              validation, and test splits of the data.
     """
     #TODO(degraff): floating point math could mess this up
-    if sum(sizes) != 1:
-        raise ValueError(f"train/val/test fractions must add up to 1! got: {sizes}")
+    if not (len(sizes) == 3 and np.isclose(sum(sizes), 1)):
+        raise ValueError(f"Invalid train/val/test splits! got: {sizes}")
```

Comment:
Another minor nitpick, I think this new error message is more concise but perhaps it loses some troubleshooting clarity. Same with the other change in this commit in `utils.py`. How about something like: "Expected 3 train/val/test splits to sum to 1! got: {sizes}"

Reply:
IMO anyone programming using the package has enough common sense to diagnose the error message. There was previously no error message at all and we never got questions about it

Reply:
Fair point!